openscenario/openscenario_interpreter/src/syntax/dynamic_constraints.cpp

```diff
@@ -21,12 +21,13 @@ inline namespace syntax
 {
 DynamicConstraints::DynamicConstraints(
   const Double max_acceleration, const Double max_acceleration_rate, const Double max_deceleration,
-  const Double max_deceleration_rate, const Double max_speed)
+  const Double max_deceleration_rate, const Double max_speed, const Double max_jerk)
```

Comment:
When I was developing `out_of_range` I needed a value limits for jerk.
I thought adding `maxJerk` to [DynamicConstraints.msg](https://github.com/tier4/scenario_simulator_v2/blob/d43f5dd9d7f645ab4ea1c649fc74301e84c0cf71/simulation/traffic_simulator_msgs/msg/DynamicConstraints.msg#L6) and [Performance.msg](https://github.com/tier4/scenario_simulator_v2/blob/d43f5dd9d7f645ab4ea1c649fc74301e84c0cf71/simulation/traffic_simulator_msgs/msg/Performance.msg) was necessary to keep this simply and ensure clean code.
That's why I adapted the rest of the code ([commit](https://github.com/tier4/scenario_simulator_v2/pull/979/commits/6c0b8b664a4e650c9b5039d4d9f924e286470624)).
I wasn't aware that [DynamicConstraints.msg](https://github.com/tier4/scenario_simulator_v2/blob/d43f5dd9d7f645ab4ea1c649fc74301e84c0cf71/simulation/traffic_simulator_msgs/msg/DynamicConstraints.msg#L6)  had to be compatible with [DynamicConstraints](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/DynamicConstraints.html)
Of course, I will revert all `maxJerk/max_jerk` changes,
Please tell me how can i provide `jerk` limits for `out_of_range` - because in that case i have no idea.